open-craft/cert-manager
# [SE-3182] Remove certbot & acme from Pipfile.

This removes certbot from pipenv and uses the system-provided certbot instead. This change is required because we were running into compatibility issues between the version of certbot that was pinned in the pipenv (used for requesting new certificates) and the system-level certbot installation (used for renewing certificates).

See [SE-3182](https://tasks.opencraft.com/browse/SE-3182) for more info.

**Test instructions**

This change was deployed to `cert-manager-stage.net.opencraft.hosting` together with https://github.com/open-craft/ansible-playbooks/pull/181.

Create a new instance on https://stage.manage.opencraft.com/ and spawn a new appserver. Verify that a certificate for your instance gets provisioned successfully (note that Ocim stage uses the staging letsencrypt environment, so you will see an "unknown issuer" warning in your browsers).

**Reviewers**:
